### PR TITLE
fix: crashing of the app when the orientation changes

### LINKED ISSUE #1909: App Crashes when the orientation changes to landscape
**Summary:**

App crashes as soon as the orientation changes to Landscape 

**Steps to reproduce:**

- open the app
- rotate the home screen to landscape

**Expected behaviour:**

Layout should have been working properly even if the orientation was changes

**Observed behaviour:**

App Crashes when changed to landscape

https://user-images.githubusercontent.com/90026952/217056176-959c0dd2-d648-491b-901e-249f14aa543a.mp4

**Device and Android version:**

Device: Samsung M11
Android: 11

### PR DESCRIPTION
**Pull request by PratyushSingh07**
Fixes #1909
I have attached a video that demonstrates the changes in the app 
https://user-images.githubusercontent.com/90026952/217059612-00779069-a707-477a-aec8-9f804dca0aa9.mp4

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -8,6 +8,7 @@
     <uses-permission android:name="android.permission.READ_PHONE_STATE"/>
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
+    <uses-permission android:name="android.permission.USE_BIOMETRIC"/>
```
Comment (jawidMuhammadi) [inline app/src/main/AndroidManifest.xml]: Why this permission? 🤔

**Review by jawidMuhammadi — CHANGES_REQUESTED** [review]

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (PratyushSingh07) [reply app/src/main/AndroidManifest.xml]: pardon my naivete , this wasn't supposed to be pushed in this PR :( I will fix it

**Comment by PratyushSingh07** [comment]
@jawidMuhammadi I have fixed the changes, kindly review.

**Comment by jawidMuhammadi** [comment]
Add the `Fix` prefix to your PR and solve the workflow issues, please

**Comment by PratyushSingh07** [comment]
> Add the `Fix` prefix to your PR and solve the workflow issues, please

the workflow issues mentions this `Execution failed for task ':app:checkDebugAarMetadata'.`  as the reason. It seems that these are warning messages that indicate that the project's dependencies require a higher version of the Android SDK than the one that is currently being used. But you have already opened the issue #1945 , so should i change the version of the SDK in this PR or wait for PR #1951 to be merged ? 
This is the reason that I have deduced for the workflows failing. Kindly correct me if I am wrong

**Comment by PratyushSingh07** [comment]
i have fixed this PR @jawidMuhammadi , Kindly review this :)

**Review by jawidMuhammadi — APPROVED** [review]
LGTM!

**Pull request merged**